### ISSUE TITLE
USWDS-Site - Enhancement: Update race and ethnicity pattern based on SPD 15

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds-site#2598]
### Is your feature request related to a problem? Please describe.

On March 28, 2024, OMB released updates to [Statistical Policy Directive 15: Standards for Maintaining, Collecting, and Presenting Federal Data on Race and Ethnicity](https://spd15revision.gov/). ([Press release here](https://www.whitehouse.gov/omb/briefing-room/2024/03/28/omb-publishes-revisions-to-statistical-policy-directive-no-15-standards-for-maintaining-collecting-and-presenting-federal-data-on-race-and-ethnicity/).) We need to update our pattern guidance to make sure we're in alignment with this new statistical guidance.

The key revisions are these:

> - Using one combined question for race and ethnicity, and encouraging respondents to select as many options as apply to how they identify.
> - Adding Middle Eastern or North African as a new minimum category. The new set of minimum race and/or ethnicity categories are:
>   - American Indian or Alaska Native
>   - Asian
>   - Black or African American
>   - Hispanic or Latino
>   - Middle Eastern or North African
>   - Native Hawaiian or Pacific Islander
>   - White
> - Requiring the collection of additional detail beyond the minimum required race and ethnicity categories for most situations, to ensure further disaggregation in the collection, tabulation, and presentation of data when useful and appropriate.

## Combined question

Respondents shall be offered a single combined race and ethnicity question that allows them to select one category or multiple categories [[Fed Register reference, p-185](https://www.federalregister.gov/d/2024-06469/p-185)]

## Detailed categories

**Miniumum categories and required text:** Respondents must be offered the following detailed categories for the corresponding minimum categories [[Fed Register reference, p-186](https://www.federalregister.gov/d/2024-06469/p-186)]

```
Asian:
Chinese, Asian Indian, Filipino, Vietnamese, Korean, and Japanese, Another group (for example, Pakistani, Hmong, Afghan, etc.)

Black or African American:
African American, Jamaican, Haitian, Nigerian, Ethiopian, Somali, Another group (for example, Trinidadian and Tobagonian, Ghanian, Congolese, etc.)

Hispanic or Latino:
Mexican, Puerto Rican, Salvadoran, Cuban, Dominican, Guatemalan, Another group (for example, Colombian, Honduran, Spaniard, etc.)

Middle Eastern or North African:
Lebanese, Iranian, Egyptian, Syrian, Iraqi, Israeli, Another group (for example, Moroccan, Yemeni, Kurdish, etc.)

Native Hawaiian or Pacific Islander:
Native Hawaiian, Samoan, Chamorro, Tongan, Fijian, Marshallese, Another group (for example, Chuukese, Palauan, Tahitian, etc.)

White:
English, German, Irish, Italian, Polish, Scottish, Another group (for example, French, Swedish, Norwegian, etc.)
```

**Write-in fields to allow self-identification:** Whenever possible, the “Another group” detail category checkboxes should be replaced with write-in fields that allows respondents to self-identify [[Fed Register reference, p-193](https://www.federalregister.gov/d/2024-06469/p-193)]

The instructions for the write-in boxes should read `Enter, for example,` followed by the examples listed in parentheses above. For the American Indian or Alaska Native category, the instructions for the write-in option should read:

> Enter, for example, Navajo Nation, Blackfeet Tribe of the Blackfeet Indian Reservation of Montana, Native Village of Barrow Inupiat Traditional Government, Nome Eskimo Community, Aztec, Maya, etc.

## Instructions for questions

The question instructions will vary depending on whether there is a write-in field or if there are detailed categories. [[Fed Register reference, p-195](https://www.federalregister.gov/d/2024-06469/p-195)] For questions with detailed categories and no write-in fields, the question instructions should read:

> What is your race and/or ethnicity? Select all that apply.

When write-in fields are provided, the instructions should read:

> What is your race and/or ethnicity? Select all that apply and enter additional details in the spaces below.

When collecting only the minimum categories, the question instructions should read:

> What is your race and/or ethnicity? Select all that apply.

## Examples

> When using the minimum categories only, the quality of the data and consistency with other datasets may be improved by providing the respondent with examples as shown in Figure 2. Agencies should provide these examples when feasible over the example in Figure 3 without examples.

[[Fed Register reference, p-196](https://www.federalregister.gov/d/2024-06469/p-196)]
[[Fed Register reference, p-198](https://www.federalregister.gov/d/2024-06469/p-198)]

### Figure 1: Race and ethnicity question with minimum categories, multiple detailed checkboxes, and write-in response area with example groups:

<img width="522" alt="Screenshot 2024-04-03 at 10 01 26 AM" src="https://github.com/uswds/uswds-site/assets/11464021/ea15b5ef-6d48-4dcb-a64f-6c82a2c0e452">

### Figure 2: Race and ethnicity question with minimum categories only, and examples:

<img width="538" alt="Screenshot 2024-04-03 at 10 01 39 AM" src="https://github.com/uswds/uswds-site/assets/11464021/450e3d96-e554-4a6e-a297-b0093078c32a">

### Figure 3: Race and ethnicity question with minimum categories only:

<img width="431" alt="Screenshot 2024-04-03 at 10 01 52 AM" src="https://github.com/uswds/uswds-site/assets/11464021/543de454-0643-469e-a23c-3f7e938677dc">

### Describe the solution you'd like

We'll need to look at our existing guidance and understand any gaps with the revised SPD 15. The new model in SPD 15 allows for a high degree of expressiveness, which we've prioritized in patterns like this one. Our interest in expressiveness led us to two suggestions for collecting this type of data in our current guidance: an expressive open text input for nonstatistical data collection; and pointing to the old SPD 15 for statistical data collection. 

There could be an opportunity here to reconcile these two suggestions, now that the SPD 15 guidance is more expressive.

And regardless of our specific pattern guidance, we should be showing how to implement the guidance in SPD 15 using USWDS components and code. 

- [ ] Crosswalk analysis between the revised SPD 15 and our current race and ethnicity pattern guidance
- [ ] Update guidance to address the updates in SPD 15
- [ ] Provide examples of the new collection options using USWDS components and code
- [ ] Talk about what we've done with the USWDS community

### Additional context

Related to, and may supersede:

- https://github.com/uswds/uswds/issues/5762
- https://github.com/uswds/uswds/issues/5804
- https://github.com/uswds/uswds/issues/5805
- https://github.com/uswds/uswds/issues/5806
- https://github.com/uswds/uswds/issues/5808

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate feature requests.

**Comment by finekatie** [comment]
5/6: Analysis/comparison draft in progress.

from #5806 : 4/24: I have started a draft doc for the main race and ethnicity pattern page: https://docs.google.com/document/d/1iAJkHR-rPvx_97zY9S4TtMm8bAQFqAU52I_S9o3sZTA. When we have kickoff meeting, this can be discussed.

**Comment by finekatie** [comment]
[Crosswalk analysis of pattern page](https://docs.google.com/document/d/1iAJkHR-rPvx_97zY9S4TtMm8bAQFqAU52I_S9o3sZTA) is waiting for FFR before being handed off to stakeholders for approval.

**Comment by annepetersen** [comment]
FFR complete: I just passed it along to our next reviewer

**Comment by finekatie** [comment]
OMB and stakeholders have reviewed draft and provided feedback. I have incorporated into draft with notes. Anne has opened discussion with GSA OES to clarify language around ethnicity and gender. Awaiting discussion/decision and feedback.

**Comment by finekatie** [comment]
This is now in FFR with Anne and Dan.

**Comment by annepetersen** [comment]
Sent to our OMB/OIRA contact

**Comment by finekatie** [comment]
Hi @annepetersen , should this be put on hold? Backlog?

**Comment by annepetersen** [comment]
Blocked

**Comment by dct2023** [comment]
@annepetersen Can you share what is blocking this? I want to make sure I know when to follow up on this and how I can unblock.

**Comment by annepetersen** [comment]
Unlikely you'll have any impact here, @dct2023 — this is entirely fed-side, including very slow consultation with OIRA. Could check in like once a month, if that, honestly

**Comment by dct2023** [comment]
@annepetersen Thank you for the note! Can I remove it from this sprint and remove Katie from the assignee? I would suggest potentially removing all assignees and we can create a 'Blocked/On Hold' table view so we can keep track of these and it won't impact velocity as these just need to be reviewed until they are ready. Let me know what you think!

cc: @mejiaj @jdotyoon

**Comment by annepetersen** [comment]
That makes sense to me, thanks
